backend/research/models.py

```diff
@@ -32,3 +32,41 @@ class ResearchScholar(Student):
     class Meta:
         verbose_name = "research scholar"
         default_related_name = "scholars"
+
+
+class Publication(models.Model):
+    CONFERENCE = "conference"
+    JOURNAL = "journal"
+    PUB_TYPE_CHOICES = [
+        (CONFERENCE, "conference"),
+        (JOURNAL, "journal"),
+    ]
+    COMMUNICATED = "communicated"
+    REJECTED = "rejected"
+    ACCEPTED = "accepted"
+    STATUS_CHOICES = [
+        (COMMUNICATED, "communicated"),
+        (REJECTED, "rejected"),
+        (ACCEPTED, "accepted"),
+    ]
+    author_faculty = models.ManyToManyField(Faculty)
+    author_scholar = models.ManyToManyField(ResearchScholar)
+    title = models.CharField(max_length=100)
+    pub_type = models.CharField(max_length=11, choices=PUB_TYPE_CHOICES)
+    pub_date = models.DateField()
+    status = models.CharField(max_length=20, choices=STATUS_CHOICES)
+    dol_number = models.IntegerField()
+    page_number = models.IntegerField()
+
+
+class Project(models.Model):
+    proposer = models.ManyToManyField(Faculty)
+    author = models.ManyToManyField(ResearchScholar)
+    agency = models.CharField(max_length=100)
+    title = models.CharField(max_length=100)
+    pi_copi = models.CharField(max_length=100)
+    date_of_sending = models.DateField()
+    start_date = models.DateField()
+    end_date = models.DateField()
+    status = models.TextField()
+
```

Comment:
The relevant fields should be `null=True, blank=True`, as they wouldn't be available at the time of creation. Eg: `dates`